### PR TITLE
Newframework

### PR DESCRIPTION
**Pull request by chen2181**
use new framework for air quality monitoring

### CODE REVIEW

File: airquality_dc/Dockerfile
Comment (Greg-Hawkridge) [inline airquality_dc/Dockerfile:1]: Update Solution name and copyright year

File: logging/setup_logging.sh
Comment (Greg-Hawkridge) [inline logging/setup_logging.sh:1]: We can remove the logging folder from individual solutions - this is tracked in a separate repository now

File: setup_commands.txt
Comment (Greg-Hawkridge) [inline setup_commands.txt:1]: I also think it is worth removing the setup_commands.txt as we are no longer maintaining it

File: logging/setup_logging.sh
Comment (Greg-Hawkridge) [inline logging/setup_logging.sh:1]: We can stop including the logging file in individal solutions - it is tracked in it's own repository now

File: airquality_dc/Dockerfile
Comment (Greg-Hawkridge) [inline airquality_dc/Dockerfile:1]: Update Solution name and copyright year

File: timeseries_sds/app.yml
Comment (Greg-Hawkridge) [inline timeseries_sds/app.yml:1]: Add 
`- DOCKER_INFLUXDB_INIT_RETENTION=2y`
to set influx to default retention of two years

**Review by Greg-Hawkridge — CHANGES_REQUESTED** [review]

File: timeseries_sds/app.yml
Comment (chen2181) [reply timeseries_sds/app.yml:1]: done,. worth double check if this is in the right place,  cheers

**Review by Greg-Hawkridge — APPROVED** [review]

**Pull request merged**